pkg/plugins/crypto/main/example.go

```diff
@@ -0,0 +1,33 @@
+package main
```

Comment:
what's the purpose of the package?

if it is useful, should this be in some path under cmd?

Reply:
just a Tink example without a KMS set.. will remove

Reply:
removed